crepecake/src/main/java/com/google/cloud/tools/crepecake/http/ConnectionFactory.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.crepecake.http;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+class ConnectionFactory {
```

Comment:
I'm not sure this class does enough to be worth its weight.

Reply:
This class is only for testing purposes since URL is final and can't be mocked.